libgrust/libproc_macro/registration.h

```diff
@@ -25,13 +25,16 @@
 
 #include <string>
 #include "tokenstream.h"
+#include "bridge.h"
 
 namespace ProcMacro {
 
 using from_str_function_t = ProcMacro::TokenStream (*) (std::string &, bool &);
+using is_available_t = ProcMacro::BridgeState;
```

Comment:
I'm not sure we really need this `using` directive. Why not directly use `ProcMacro::BridgeState`?

Reply:
You're right, this line is a remnant from a first version of the code which used a function pointer much like from_str callback instead of a value.

Reply:
Fixed in [093ac28](https://github.com/Rust-GCC/gccrs/pull/2475/commits/093ac288152f11ed18391b19d820bc7ee5ef0414).